accounts/src/main/java/eu/kevin/accounts/accountlinking/AccountLinkingFragment.kt

```diff
@@ -40,4 +41,9 @@ internal class AccountLinkingFragment : BaseFragment<AccountLinkingState, Accoun
     override fun onAuthorizationReceived(uri: Uri) {
         viewModel.intents.trySend(HandleAuthorization(uri))
     }
+
+    override fun handleUri(uri: Uri) {
+        val intent = Intent(Intent.ACTION_VIEW, uri)
```

Comment:
for additional safety we need to check whether the intent is handled as there might be some special cases when user won't have mail app or even phone app and intent won't be executed as well as SDK will crash